docs/config/server-options.md

```diff
@@ -318,3 +318,22 @@ export default defineConfig({
   },
 })
 ```
+
+## server.sourcemapIgnoreList
+
+- **Type:** `false | (sourcePath: string, sourcemapPath: string) => boolean`
+- **Default:** `(sourcePath) => sourcePath.includes('node_modules')`
+
+Whether or not to ignore source files in the server sourcemap, used to populate the [`x_google_ignoreList` source map extension](https://developer.chrome.com/blog/devtools-better-angular-debugging/#the-x_google_ignorelist-source-map-extension).
```

Comment:
@bmeurer not blocking for the PR, but it would be nice to be able to link to a more focused blog post about `x_google_ignoreList`. If your team ends up having a better resource in the future, would you do a PR to rewire these references?

Reply:
Thanks @jecfish, sounds great!